### PR TITLE
fix(betterUptime): make better uptime on success

### PR DESCRIPTION
**Pull request by kishore03109**
## Problem
Currently our better uptime is done on create, but it should be on success instead. 

Closes [insert issue #]

## Solution

shifting the creation of this to post success, not during the launch process. 

**Breaking Changes**

- [ ] Yes - this PR contains breaking changes
  - Details ...
- [X] No - this PR is backwards compatible with ALL of the following feature flags in this [doc](https://www.notion.so/opengov/Existing-feature-flags-518ad2cdc325420893a105e88c432be5)

Tests

- [ ] create site alunch in staging, notice that better uptime monitor was [created](https://opengovproducts.slack.com/archives/C04RDJ504DC/p1704255875013479)

### CODE REVIEW

File: src/routes/formsg/formsgSiteLaunch.ts
Comment (alexanderleegs) [inline src/routes/formsg/formsgSiteLaunch.ts:1]: We're still attempting to create the monitor in lines 310-311
(```
// Create better uptime monitor
await this.createMonitor(launchResult.value.primaryDomainSource)```)
,

this should be removed right!

Comment (kishore03109) [reply src/routes/formsg/formsgSiteLaunch.ts:1]: 59ab717d8de7899bb18feae6465f414a801cf0bd

Comment (kishore03109) [reply src/routes/formsg/formsgSiteLaunch.ts:1]: ps forgot to remove

**Review by alexanderleegs — APPROVED** [review]
lgtm

**Pull request merged**